gcloud/logging/config_service_v2_api.py

```diff
@@ -14,64 +14,123 @@
 # and updates to that file get reflected here through a refresh process.
 # For the short term, the refresh process will only be runnable by Google engineers.
-# Manual additions are allowed because the refresh process performs
-# a 3-way merge in order to preserve those manual additions. In order to not
-# break the refresh process, only certain types of modifications are
-# allowed.
 #
-# Allowed modifications:
-# 1. New methods (these should be added to the end of the class)
-#
-# Happy editing!
+# The only allowed edits are to method and file documentation. A 3-way
+# merge preserves those additions if the generated source changes.
+
+import os
+import platform
 
 from google.gax import api_callable
-from google.gax import api_utils
-from google.gax import page_descriptor
+from google.gax import config
+from google.gax.path_template import PathTemplate
+import google.gax
+import yaml
+
 from google.logging.v2 import logging_config_pb2
 
 
 class ConfigServiceV2Api(object):
+    """See src/api/google/logging/v2/logging.proto for documentation"""
 
-    # The default address of the logging service.
-    _SERVICE_ADDRESS = 'logging.googleapis.com'
+    _CODE_GEN_NAME_VERSION = 'gapic-0.1.0'
 
-    # The default port of the logging service.
-    _DEFAULT_SERVICE_PORT = 443
+    SERVICE_ADDRESS = 'logging.googleapis.com'
+    """The default address of the service."""
+
+    DEFAULT_SERVICE_PORT = 443
+    """The default port of the service."""
 
     # The scopes needed to make gRPC calls to all of the methods defined in
     # this service
-    _ALL_SCOPES = (
-        'https://www.googleapis.com/auth/logging.write',
-        'https://www.googleapis.com/auth/logging.admin',
-        'https://www.googleapis.com/auth/logging.read',
-        'https://www.googleapis.com/auth/cloud-platform.read-only',
-        'https://www.googleapis.com/auth/cloud-platform',
-    )
-
-    _LIST_SINKS_DESCRIPTOR = page_descriptor.PageDescriptor(
-        'page_token',
-        'next_page_token',
-        'sinks',
-    )
-
-    def __init__(
-            self,
-            service_path=_SERVICE_ADDRESS,
-            port=_DEFAULT_SERVICE_PORT,
-            channel=None,
-            ssl_creds=None,
-            scopes=_ALL_SCOPES,
-            is_idempotent_retrying=True,
-            max_attempts=3,
-            timeout=30):
-        self.defaults = api_callable.ApiCallableDefaults(
-            timeout=timeout,
-            max_attempts=max_attempts,
-            is_idempotent_retrying=is_idempotent_retrying)
-        self.stub = api_utils.create_stub(
+    _ALL_SCOPES = ('https://www.googleapis.com/auth/logging.write',
+                   'https://www.googleapis.com/auth/logging.admin',
+                   'https://www.googleapis.com/auth/logging.read',
+                   'https://www.googleapis.com/auth/cloud-platform.read-only',
+                   'https://www.googleapis.com/auth/cloud-platform', )
```

Comment:
We checked that this was valid on pylint and pep8.

FYI: Our code-gen pipeline is currently using python reformatter as a final step. 
This stays as is, unless you can provide links to a better re-formatting tool that improves this.